src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/SendNotificationHandler.java

```diff
@@ -126,7 +130,8 @@ private List<Event> getEventsToHandle(Callback<AsylumCase> callback) {
         }
 
         // TODO Get rid of this when notification is activated for AIP
-        if (HandlerUtils.isAipJourney(callback.getCaseDetails().getCaseData())) {
+        if (HandlerUtils.isAipJourney(callback.getCaseDetails().getCaseData())
+            && !featureToggler.getValue("aip-ftpa-feature", false)) {
```

Comment:
You might want to remove the comment here and maybe only disable the notification if feature flag returns true, that is the if condition here would only consider the feature flag and not the journey type, since the related ticket is about activating AIP notification.

Reply:
I think both conditions still apply because APPLY_FOR_FTPA_APPELLANT triggers notifications both in the LR and AIP journey.

If we remove APPLY_FOR_FTPA_APPELLANT from the qualifying events based only on the flag, then when the flag is off all notifications will be switched off (including the ones in the LR journey, currently being sent to AO, LR and HO). With both conditions we can remove/keep the event based on the flag but only scoped to the AIP journey, so the LR journey stays untouched.

Reply:
Yes you are very correct. Thanks.